xcatlkd/happygram
# Bugfix

This commit should resolve the issue of sessions not persisting after redirect.
This also makes the req.user object available to all routes.

To test:
launch server and navigate to /signup
create a new user (login form coming any day now.)
this should redirect to the "home" page and display user info.
Can then navigate to /form page and submit a photo
